tests/test_scalars.py

```diff
@@ -156,116 +181,131 @@ def test_timedelta_add_sub() -> None:
-    as5 = np.datetime64(1, "ns")
-    as6 = dt.timedelta(days=1)
-    as7 = np.timedelta64(1, "D")
-    as8 = pd.TimedeltaIndex([td])
-    as9 = pd.Series([td])
-    as10 = pd.period_range("2012-01-01", periods=3, freq="D")
-    as11 = pd.date_range("2012-01-01", periods=3)
-    as12 = ndarray_td64
-    as13 = ndarray_dt64
-    as14 = pd.NaT
+    as_period = pd.Period("2012-01-01", freq="D")
+    as_timestamp = pd.Timestamp("2012-01-01")
+    as_datetime = dt.datetime(2012, 1, 1)
+    as_date = dt.date(2012, 1, 1)
+    as_datetime64 = np.datetime64(1, "ns")
+    as_dt_timedelta = dt.timedelta(days=1)
+    as_timedelta64 = np.timedelta64(1, "D")
+    as_timedelta_index = pd.TimedeltaIndex([td])
+    as_timedelta_series = pd.Series(as_timedelta_index)
+    as_period_index = pd.period_range("2012-01-01", periods=3, freq="D")
+    as_datetime_index = pd.date_range("2012-01-01", periods=3)
+    as_ndarray_td64 = ndarray_td64
+    as_ndarray_dt64 = ndarray_dt64
+    as_nat = pd.NaT
 
     check(assert_type(td + td, pd.Timedelta), pd.Timedelta)
-    check(assert_type(td + as1, pd.Period), pd.Period)
-    check(assert_type(td + as2, pd.Timestamp), pd.Timestamp)
-    check(assert_type(td + as3, pd.Timestamp), pd.Timestamp)
-    check(assert_type(td + as4, dt.date), dt.date)
-    check(assert_type(td + as5, pd.Timestamp), pd.Timestamp)
-    check(assert_type(td + as6, pd.Timedelta), pd.Timedelta)
-    check(assert_type(td + as7, pd.Timedelta), pd.Timedelta)
-    check(assert_type(td + as8, pd.TimedeltaIndex), pd.TimedeltaIndex)
-    check(assert_type(td + as9, pd.Series), pd.Series)
-    check(assert_type(td + as10, pd.PeriodIndex), pd.PeriodIndex)
-    check(assert_type(td + as11, pd.DatetimeIndex), pd.DatetimeIndex)
-    check(assert_type(td + as12, npt.NDArray[np.timedelta64]), np.ndarray)
+    check(assert_type(td + as_period, pd.Period), pd.Period)
+    check(assert_type(td + as_timestamp, pd.Timestamp), pd.Timestamp)
+    check(assert_type(td + as_datetime, pd.Timestamp), pd.Timestamp)
+    check(assert_type(td + as_date, dt.date), dt.date)
+    check(assert_type(td + as_datetime64, pd.Timestamp), pd.Timestamp)
+    check(assert_type(td + as_dt_timedelta, pd.Timedelta), pd.Timedelta)
+    check(assert_type(td + as_timedelta64, pd.Timedelta), pd.Timedelta)
+    check(assert_type(td + as_timedelta_index, pd.TimedeltaIndex), pd.TimedeltaIndex)
+    check(assert_type(td + as_timedelta_series, TimedeltaSeries), pd.Series)
+    check(assert_type(td + as_period_index, pd.PeriodIndex), pd.PeriodIndex)
+    check(assert_type(td + as_datetime_index, pd.DatetimeIndex), pd.DatetimeIndex)
+    check(
+        assert_type(td + as_ndarray_td64, npt.NDArray[np.timedelta64]),
+        np.ndarray,
+        np.timedelta64,
+    )
     # pyright has trouble with timedelta64 and datetime64
     check(
         assert_type(
-            td + as13,  # pyright: ignore[reportGeneralTypeIssues]
+            td + as_ndarray_dt64,  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
can remove the `# pyright` now that pyright has a new release with the bug fixed.

Reply:
Removed.